src/DependencyInjection/SonataClassificationExtension.php

```diff
@@ -52,7 +53,10 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('admin.xml');
         }
 
-        $this->registerDoctrineMapping($config);
+        if (isset($bundles['SonataDoctrineBundle'])) {
+            $this->registerDoctrineMapping($config);
```

Comment:
I would expect an exception, if the bundle isn't loaded. Otherwise this could have some side effects if you don't create the doctrine mapping